src/unity/python/turicreate/toolkits/image_classifier/_annotate.py

```diff
@@ -0,0 +1,216 @@
+"""
+from __future__ import print_function as _
+from __future__ import division as _
+from __future__ import absolute_import as _
+
+from ...visualization import _get_client_app_path
+
+import turicreate as __tc
+
+def _warning_annotations():
+    print(
+        """
+        *** WARNING: If you don't assign the output of the annotate function
+                     to a variable all annotations are going to be lost.
+                     To recover the last annotated SFrame call:
+
+        `tc.image_classifier._recover_annotation()`
+
+        """
+    )
+
+def annotate(data, image_column = 'image', annotation_column = 'annotations'):
+    """
+        Annotate your images loaded in either an SFrame or SArray Format
+
+        The annotate util is a GUI assisted application used to create labels in
+        SArray Image data. Specifying a column, with dtype Image, in an SFrame
+        works as well since SFrames are composed of multiple SArrays.
+
+        When the GUI is terminated an SFrame is returned with the representative,
+        images and annotations.
+
+        The returned SFrame includes the newly created annotations.
+
+        Parameters
+        --------------
+        data : SArray | SFrame
+            The data containing the images. If the data type is 'SArray'
+            the 'image_column', and 'annotation_column' variables are used to construct
+            a new 'SFrame' containing the 'SArray' data for annotation.
+            If the data type is 'SFrame' the 'image_column', and 'annotation_column'
+            variables are used to annotate the images.
+
+        image_column: string, optional
+            If the data type is SFrame and the 'image_column' parameter is specified
+            then the column name is used as the image column used in the annotation. If
+            the data type is 'SFrame' and the 'image_column' variable is left empty. A
+            default column value of 'image' is used in the annotation. If the data type is
+            'SArray', the 'image_column' is used to construct the 'SFrame' data for
+            the annotation
+
+        annotation_column : string, optional
+            If the data type is SFrame and the 'annotation_column' parameter is specified
+            then the column name is used as the annotation column used in the annotation. If
+            the data type is 'SFrame' and the 'annotation_column' variable is left empty. A
+            default column value of 'annotation' is used in the annotation. If the data type is
+            'SArray', the 'annotation_column' is used to construct the 'SFrame' data for
+            the annotation
+
+
+        Returns
+        -------
+
+        out : SFrame
+            A new SFrame that contains the newly annotated data.
+
+        Examples
+        --------
+
+        >> import turicreate as tc
+        >> images = tc.image_analysis.load_images("path/to/images")
+        >> images
```

Comment:
Maybe change this line to `print(image)` to make it clear what's happening on the lines below it.